### PR TITLE
ptCaseStudy

### PR DESCRIPTION
**Pull request by burnoutcomputer**
Assignment turn in.

### CODE REVIEW

**Comment by mbMosman** [comment]
Please update your GitHub profile with your name to facilitate grading.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            12010 Pacific Trails Road<br>
+            Zephyr, CA 95555<br>
+            <br>
+            888-555-5555
+        </div>
+    </div>
+    <br>
+    <footer>
+        Copyright &copy; 2014 Pacific Trails Resort<br>
+        eric@nelson.com
```
Comment (mbMosman) [inline index.html]: The email in the footer should be an email (mailto) link.

File: yurts.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            <h3>What is a Yurt?</h3>
+            <p>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+            <h3>How are the yurts furnished?</h3>
+            <p>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove. The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</p>
+            <h3>What should I bring?</h3>
+            <p>Bring a sense of adventure and time to relax! Most guests also pack comfortable walking shoes and plan to dress for changing weather with layers of clothings.</p>
+            <br>
+        </div>
+            <footer> Copyright &copy; 2014 Pacific Trails Resort<br>
+            eric@nelson.com</footer>
```
Comment (mbMosman) [inline yurts.html]: The email in the footer should be an email (mailto) link.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    </header>
+    
+    <nav>
+        <a href="index.html">Home</a> &nbsp; 
+        <a href="yurts.html">Yurts</a> &nbsp; 
+        <a href="activitites.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a>
+    </nav>
+    
+        <div id="content">
```
Comment (mbMosman) [inline yurts.html]: This should be a `<main>` tag. This was a late update to my instructions, but please change before submitting part 2.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    </header>
+    
+    <nav>
+        <a href="index.html">Home</a> &nbsp; 
+        <a href="yurts.html">Yurts</a> &nbsp; 
+        <a href="activitites.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a>
+    </nav>
+    
+    <div id="content">
```
Comment (mbMosman) [inline index.html]: This should be a `<main>` tag. This was a late update to my instructions, but please change before submitting part 2.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<title>Pacific Trails</title>
```
Comment (mbMosman) [inline index.html]: You are missing your `<head>` and `<body>` tags.

File: yurts.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<title>Pacific Trails :: Yurts</title>
```
Comment (mbMosman) [inline yurts.html]: You are missing your `<head>` and `<body>` tags.

File: yurts.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    </nav>
+    
+        <div id="content">
+            <h2>The Yurts at Pacific Trails</h2>
+            <h3>What is a Yurt?</h3>
+            <p>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+            <h3>How are the yurts furnished?</h3>
+            <p>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove. The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</p>
+            <h3>What should I bring?</h3>
+            <p>Bring a sense of adventure and time to relax! Most guests also pack comfortable walking shoes and plan to dress for changing weather with layers of clothings.</p>
```
Comment (mbMosman) [inline yurts.html]: The text for the paragraph isn't quite accurate.  You can copy the text from the wireframes to avoid typos and errors.

**Comment by mbMosman** [comment]
Overall looks good.  Make sure to update the email link and proof read your text before submitting for part 2.

**Comment by burnoutcomputer** [comment]
Eric Nelson, enelson019@my.hennepintech.edu

I'll update GitHub later.

I had put the mailto in, but I was having issues selecting things with my
laptop, and I must have somehow deleted things. I completely forgot about
my <head> and <body> tags, though. I didn't see the <main> tag instructions.

**Pull request closed without merging**